### ISSUE TITLE
Sort by Date Added is not showing in date added order, just started today after a git pull

### ISSUE THREAD

**Issue by walt93** [WWBN/AVideo#8191]
**Describe the bug**
I have an avideo site which gets its media from a web scraper that pushes media to mobileManager/upload.php, then makes a mysql call to update the "created" and "users_id" fields. This has been working for weeks.

Today, after doing a git pull, it's no longer honoring these created dates very well. See screenshot below (expand so you can see the "created" date of the videos) and I note that clearing the cache/reloading the page didn't change the results.

What's happening is, I've just added a few new channels, so it's going through and ingesting old media and setting the "created" date to the original "published_date" of the video. The last bunch of videos it's ingested are shown on top, from 2 weeks to 7 months old, then they're followed by videos from today…

![Screen Shot 2023-07-18 at 4 54 58 PM](https://github.com/WWBN/AVideo/assets/11711050/7f9fb8a0-1716-4722-972e-1f4f27e0021a)

***** current result ******
Videos aren't displaying in reverse "created" order

***** expected result *****
Videos to display in reverse "created" order

**Comment by DanielnetoDotCom** [comment]
Please check this:
get the first video in your list in the database, check the table videos, the column order.

what value do you have there?

I tested on my side and the URL https://my.site/?showOnly=dateAddedOrder&created=ASC is sorting properly

**Comment by walt93** [comment]
I'm not tracking you Daniel, in mysql doing a "describe videos" doesn't show a column order. What should I be checking?

I'm on https://news.conspyre.tv/?showOnly=dateAddedOrder&created=asc and the items, in order are:
   12 hours ago
   5 days ago
   2 weeks ago
   2 months ago
   ...
   1 year ago
   then
   8 hours ago
   9 hours ago
    10 hours ago
    ...and so on

**Comment by walt93** [comment]
Hmm… Daniel let me try to explain again. Before the git pull there are ≈4,900 videos that all appear in the correct order. The value of 'order' for these 4.900 videos is NULL.

After the git pull, about half the uploads have an order of '0'. These videos are showing out of order.

When I went into mysql last night and manually set these to NULL, they appeared in the correct order.

If I write a cron task that goes out every 5 minutes and whacks 'order' to NULL for all videos, it will mask the bug which appears to have been recently introduced which is causing some-but-not-all of the 'order' fields to be 0.

It seems that if the order field is set to 0, the sorting behavior is different than if they are NULL.

**Comment by DanielnetoDotCom** [comment]
just confirming, if is all null it works, if is 0 it fails. is that correct?

please check this update

**Comment by walt93** [comment]
Pulled. Will report back later today to confirm fix.

**Comment by walt93** [comment]
Fix is confirmed
